src/main/java/microservices/userServiceBasic/api/mapper/UserApiMapper.java

```diff
@@ -0,0 +1,19 @@
+package microservices.userServiceBasic.api.mapper;
+
+import microservices.userServiceBasic.api.model.UserDetails;
+import microservices.userServiceBasic.domain.model.User;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserApiMapper {
+
+	public static UserDetails mapToUserDetails(User user) {
```

Comment:
This method should not be static (As I said in a previous comment).

Reply:
Having another look. You should not need this method. You only need to map UserDetails to User, not the other way around.

Reply:
Based on my design I do need both. Let's talk about this tomorrow.